### PR TITLE
feat: Add ActiveProposals component in Dashboard

### PR DESCRIPTION
**Pull request by Hemanthghs**
#406

### CODE REVIEW

File: frontend/src/pages/gov/ProposalItem.js [hunk 1]
```diff
@@ -1,55 +1,76 @@
...
+import Tooltip from "@mui/material/Tooltip";
 
 export const ProposalItem = (props) => {
-  const { info, tally, vote, poolInfo, onItemClick } = props;
+  const { info, vote, poolInfo, onItemClick, chainUrl, proposalId } = props;
+  const [pTally, setPTally] = useState([]);
+  const tally = pTally;
   const tallyInfo = computeVotePercentage(tally, poolInfo);
-
+  const tallySum =
```
Comment (charymalloju) [inline frontend/src/pages/gov/ProposalItem.js:19]: destructre the object. const {yes, no, no_with_veto, abstain } = tallyInfo;

File: frontend/src/pages/gov/ProposalItem.js [hunk 2]
```diff
@@ -1,55 +1,76 @@
...
+    no_with_veto: (tallyInfo.no_with_veto / tallySum) * 100,
+    abstain: (tallyInfo.abstain / tallySum) * 100,
+  };
   const onVoteClick = () => {
     props.setOpen(info?.proposal_id);
   };
 
+  useEffect(() => {
+    if (walletConnected) {
+      const response = async (chainUrl) => {
```
Comment (charymalloju) [inline frontend/src/pages/gov/ProposalItem.js:37]: use try and catch block

File: frontend/src/pages/gov/Proposals.js [hunk 3]
```diff
@@ -223,6 +226,38 @@ export default function Proposals() {
           }}
         />
       ) : null}
+      {proposals.length === 0 ? (
```
Comment (charymalloju) [inline frontend/src/pages/gov/Proposals.js:229]: proposals?.length ===0 and !proposals?.length both are same. But recommended to use !proposals?.length always.

File: package-lock.json
Comment (charymalloju) [inline package-lock.json:1]: remove this file

File: node_modules/vfile/readme.md
Comment (charymalloju) [inline node_modules/vfile/readme.md:1]: we should not push node modules to github

**Review by aleem1314 — APPROVED** [review]
utACK

**Pull request merged**